lib/lightning_web/controllers/api/workflows_controller.ex

```diff
@@ -0,0 +1,461 @@
+defmodule LightningWeb.API.WorkflowsController do
+  use LightningWeb, :controller
+
+  alias Ecto.Changeset
+
+  alias Lightning.Extensions.Message
+  alias Lightning.Extensions.UsageLimiting.Action
+  alias Lightning.Extensions.UsageLimiting.Context
+  alias Lightning.Graph
+  alias Lightning.Policies.Permissions
+  alias Lightning.Projects.Project
+  alias Lightning.Repo
+  alias Lightning.Services.UsageLimiter
+  alias Lightning.Workflows
+  alias Lightning.Workflows.Edge
+  alias Lightning.Workflows.Presence
+  alias Lightning.Workflows.Workflow
+  alias LightningWeb.ChangesetJSON
+
+  action_fallback LightningWeb.FallbackController
+
+  require Logger
+
+  def index(conn, %{"project_id" => project_id}) do
+    with :ok <- authorize_read(conn, project_id) do
+      list =
+        Workflows.list_project_workflows(project_id,
+          include: [:edges, :jobs, :triggers]
+        )
+
+      json(conn, %{workflows: list, errors: []})
```

Comment:
Why do we return the `errors` even when it's `200`? Does the `spec` require that it be present?

Reply:
On the collections API we where always returning the field, on the client side I think it helps to avoid an extra check if it's available and only then reading.